src/richie/apps/core/templatetags/language_name.py

```diff
@@ -0,0 +1,23 @@
+"""Custom template tags related to FEATURES settings."""
+from django import template
+from django.conf import global_settings
+from django.utils import translation
+
+# do not use the lazy
+# pylint: disable=import-outside-toplevel
```

Comment:
No more relevant as the following import is at top level

Reply:
Fixed!